src/main/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsController.java

```diff
@@ -53,4 +69,36 @@ public ResponseEntity<Void> updateAudioRequestLastAccessedTimestamp(Integer medi
         mediaRequestService.updateAudioRequestLastAccessedTimestamp(mediaRequestId);
         return new ResponseEntity<>(HttpStatus.OK);
     }
+
+    @Override
+    @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
+    @Authorisation(contextId = MEDIA_REQUEST_ID,
+        securityRoles = {TRANSCRIBER})
+    public ResponseEntity<Resource> download(Integer mediaRequestId) {
+        InputStream audioFileStream = mediaRequestService.download(mediaRequestId);
+
+        return new ResponseEntity<>(
+            new InputStreamResource(audioFileStream),
+            HttpStatus.OK
+        );
+    }
+
+    @Override
+    public ResponseEntity<AddAudioResponse> addAudioRequest(AudioRequestDetails audioRequestDetails) {
```

Comment:
Since we're touching this endpoint - would you mind adding the `@SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)` for the swagger?

Reply:
Pull request opened [#532 DMP-1168 Audio Requests swagger / postman](https://github.com/hmcts/darts-api/pull/532)